gutenberg-ramp.php

```diff
@@ -111,6 +112,23 @@ function ramp_for_gutenberg_load_gutenberg( $criteria = false ) {
 	gutenberg_ramp_load_gutenberg( $criteria );
 }
 
+/**
+ * @return bool|string
+ */
+function gutenberg_ramp_validated_gutenberg_load_path() {
```

Comment:
Minor: this would be better named `gutenberg_ramp_get_validated_gutenberg_load_path`:

```suggestion
function gutenberg_ramp_get_validated_gutenberg_load_path() {
```